low-level/files/opt/seagate/sspl/bin/conf_based_sensors_enable

```diff
@@ -1,60 +1,85 @@
-#!/bin/bash
-set -e -u -o pipefail
-
-to_lower_case() {
-    tr '[:upper:]' '[:lower:]'
-}
-
-declare -a sed_expr=(-e '/^\[[A-Z]\+\]$/h')
-storage_type="$(provisioner get_setup_info --output json | jq -r .ret.storage_type | to_lower_case)"
-server_type="$(provisioner get_setup_info --output json | jq -r .ret.server_type | to_lower_case)"
-
-declare -A monitor=(
-    [REALSTORSENSORS]="true"
-    [NODEHWSENSOR]="true"
-    [SYSTEMDWATCHDOG]="true" # This is the 'server disks' sensor
-    [RAIDSENSOR]="true"
-    [SASPORTSENSOR]="true"
-    [MEMFAULTSENSOR]="true"
-    [CPUFAULTSENSOR]="true"
-
-)
-
-case "${storage_type}" in
-    virtual) 
-        monitor[REALSTORSENSORS]="false"
-        ;;
-    jbod)
-        monitor[REALSTORSENSORS]="false"
-        ;;
-esac
-
-case "${server_type}" in
-    virtual)
-        monitor[NODEHWSENSOR]="false"
-        monitor[SASPORTSENSOR]="false"
-        monitor[MEMFAULTSENSOR]="false"
-        monitor[CPUFAULTSENSOR]="false"
-        monitor[RAIDSENSOR]="false"
-        ;;
-esac
-
-for group in "${!monitor[@]}"; do
-    # We remove comments after the value since those are improperly parsed by the ConfigReader
-    # in sspl_test.
-    sed_expr+=(-e '/^monitor=.*$/{ x ; /^\['"${group}"'\]$/{ x ; s/^monitor=.*$/monitor='"${monitor[${group}]}"'/;  b} ; x }')
-done
-
-
-[ -f "${SSPL_CONF}" ] && {
-    sed -i "${sed_expr[@]}" $SSPL_CONF
-}
-for group in "${!monitor[@]}"; do
-    ${CONSUL_PATH}/consul kv put sspl/config/"${group}"/monitor "${monitor["${group}"]}"
-done
-
-file_store_config_path="/opt/seagate/${PRODUCT_FAMILY}/sspl/sspl_test/conf/sspl_tests.conf"
-[ -f "${file_store_config_path}" ] && {
-    sed -i "${sed_expr[@]}" $file_store_config_path
-}
+#!/usr/bin/python3.6
 
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+import configparser
+import sys
+import os
+
+# Add the top level directories
+sys.path.insert(0, f'/opt/seagate/cortx/sspl/low-level')
```

Comment:
TODO: insert path by putting it in a constant and then using it

Reply:
This will be addressed in EOS-16669